lib/rules/no-private-routing-service.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const emberUtils = require('../utils/ember');
+const types = require('../utils/types');
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const ERROR_MESSAGE =
+  "Don't inject the private '-routing' service. Instead use the public 'router' service.";
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'disallow injecting the private routing service',
+      category: 'Best Practices',
+      recommended: true,
```

Comment:
New rules should actually not be added as `recommended` since that's a breaking change. In a future major version bump, we will likely make this rule `recommended` :)

Reply:
Makes sense. Reverted that bit